<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Safari_531227_Mac_OS.AssertExceptionTest" errors="0" failures="0" tests="9" time="0.0030">
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should pass when callback throws exception" time="0.0010"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should pass with message when callback throws exception" time="0.0"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should pass when callback throws correct exception" time="0.0"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should pass with message when callback throws correct exception" time="0.0"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should fail when callback does not throw exception" time="0.0"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should fail with message when callback does not throw exception" time="0.0010"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should fail with message and exception type when callback does not throw exception" time="0.0"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should fail when exception thrown is not of correct type" time="0.0"/>
<testcase classname="Safari_531227_Mac_OS.AssertExceptionTest" name="test should fail with message when exception thrown is not of correct type" time="0.0010"/>
</testsuite>
